<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="leIpcName">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>deeplink_sdk</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbtnConnect">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect IPC</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>53</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>auth id</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leAuthId">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>deeplink_test</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>auth token</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leAuthToken">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>60</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0123456789</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbtnAuth">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>60</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Auth</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ipc name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>53</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>device id</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>100</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>device password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leDeviceId">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leDevicePassword">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>100</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbtnOpenRemote">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>100</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>OpenRemote</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbtnCloseRemote">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>100</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>CloseRemote</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbtnGetDeviceInfo">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>60</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>GetDeviceInfo</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="teMessage">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>861</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ipc name of remote video proccess</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leRemoteIpcName">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>140</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbtnConnectRemoteIpc">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>140</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chbFullScreen">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>100</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>full screen</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>remote controlled by device id</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leRemoteControlledById">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>170</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbtnCloseRemoteControlled">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>170</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbtnSetConnectOption">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>140</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>SetConnectOption</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbtnSendData">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>140</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>SendData</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chbRemoteControlledSwitch">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>20</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>remote controlled prompt switch</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbtnSendData2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>170</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>SendData</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chbMicrophone">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>100</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Microphone</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
